### ISSUE TITLE
Invoices should list the ticket IDs

### ISSUE THREAD

**Issue by malemburg** [EuroPython/epcon#1264]
This would help with badge pickups, since attendees often bring their invoices as proof.

Invoice currently just list the ticket code:

![image](https://user-images.githubusercontent.com/6735498/79550065-260d1f80-8098-11ea-84f9-b6520262f3c4.png)

**Comment by matrixise** [comment]
Good point, for EP2021, we will some people with the invoice but not with the ticket number.

**Comment by malemburg** [comment]
Would be nice to have to EP2020 already, since we'll again use the ticket ID on the (virtual) registration desk.

**Comment by fpierfed** [comment]
The Invoice class is in assopy/models.py and already imports Ticket from conference/models.py

**Comment by ethancarlsson** [comment]
I'll get started on this one
